paddle/framework/CMakeLists.txt

```diff
@@ -44,5 +44,12 @@ add_custom_command(TARGET framework_py_proto POST_BUILD
 cc_library(backward SRCS backward.cc DEPS net_op)
 cc_test(backward_test SRCS backward_test.cc DEPS backward recurrent_op device_context)
 
+cc_library(executor SRCS executor.cc DEPS op_registry device_context scope framework_proto ${GLOB_OP_LIB})
+if(WITH_GPU)
+    nv_test(executor_test SRCS executor_test.cc DEPS executor)
```

Comment:
Should we run `cc_test(executor_test SRCS executor_test.cc DEPS executor)` as well when WITH_GPU is ON?

Reply:
When WITH_GPU is ON, both cpu and gpu code will be tested.